src/agent/protocols/protos/agent.proto

```diff
@@ -67,7 +67,6 @@ service AgentService {
 	rpc CopyFile(CopyFileRequest) returns (google.protobuf.Empty);
 	rpc GetOOMEvent(GetOOMEventRequest) returns (OOMEvent);
 	rpc AddSwap(AddSwapRequest) returns (google.protobuf.Empty);
-	rpc PullImage(PullImageRequest) returns (google.protobuf.Empty);
```

Comment:
I think we will also need to update the virtcontainers agent code to have it use the new Image service as well?

Reply:
You said is this: https://github.com/kata-containers/kata-containers/pull/2956/files#diff-f87c1c212bba3abd0b07bafe69d5b01ad00dfe2c01bed585e2a75fcbb6442b89L681

Reply:
No, I mean that virtcontainers won't be able to reach the new agent Image service API. I think you need to add something like https://github.com/kata-containers/kata-containers/pull/2733/files#diff-6fb7f551035031ed29ad100c61f1a7e92b0fa15d2f67e4c7ea7ebf3b715583f4 to this PR.

Reply:
oh, I got, coming soon.

Reply:
Thanks.